pkg/operator/ceph/csi/spec.go

```diff
@@ -313,3 +319,53 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 	}
 	return nil
 }
+
+// ValidateCSIVersion checks if the configured ceph-csi image is supported
+func ValidateCSIVersion(namespace string, clientset kubernetes.Interface, csiPluginImage string, rookImage string, serviceAccountName string) (bool, error) {
```

Comment:
`namespace string, clientset kubernetes.Interface, csiPluginImage string, rookImage string, serviceAccountName string`
to 
`namespace  , csiPluginImage , rookImage , serviceAccountName string ,clientset kubernetes.Interface`

Reply:
Nit: I'd put clientset first